moped-editor/src/components/GridTable/GridTableFilters.js

```diff
@@ -520,6 +520,7 @@ const GridTableFilters = ({
                       handleFilterOperatorClick(filterId, e.target.value)
                     }
                     label="field"
+                    data-testid="operator-select"
```

Comment:
Finding the right part of some of the MUI components to interact was tricky so these props gives us a way to find them using [ByTestId](https://testing-library.com/docs/queries/bytestid)